config-example.sh

```diff
@@ -57,8 +57,9 @@ export DEFAULTFOURDRIVESWRAIDLEVEL="6"
 export DEFAULTLVM="0"
 export DEFAULTLOADER="grub"
 export DEFAULTGOVERNOR="ondemand"
-
+export FORCE_GPT=1
```

Comment:
should we start documenting at least new options in the README.md?

Reply:
```suggestion
declare -x -i FORCE_GPT=1
``` seems to be nicer if this stays an integer always (which it does according to commit message)

Reply:
> should we start documenting at least new options in the README.md?

sounds good, I have started a new subsection in the readme beginning with this DWC_ENABLE setting.

Reply:
> Suggested change Beta 
> [Give us feedback](https://www.research.net/r/SuggestedChanges)
> export FORCE_GPT=1
> declare -x -i FORCE_GPT=1
> Commit suggestion  Add suggestion to batch
> seems to be nicer if this stays an integer always (which it does according to commit message)

great suggestion but a bit out of scope for this PR